src/entities/day.ts

```diff
@@ -49,26 +64,33 @@ export const weekMap: WeekMap = {
   Sat: "土",
   Sun: "日",
 };
-export type DayJa = "月" | "火" | "水" | "木" | "金" | "土";
-
-export const dayJaList: DayJa[] = ["月", "火", "水", "木", "金"];
-export const weeks: Day[] = ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
-export type SpecialDay = "Intensive" | "Appointment" | "AnyTime";
-export type SpecialDayJa = "集中" | "応談" | "随時";
-export const specialDays: SpecialDay[] = [
-  "Intensive",
-  "Appointment",
-  "AnyTime",
-];
-export const specialDayMap: { [key in SpecialDay]: SpecialDayJa } = {
+export const specialDayMap: Record<SpecialDay, SpecialDayJa> = {
+  Intensive: "集中",
+  Appointment: "応談",
+  AnyTime: "随時",
+};
+export const fullDayMap: Partial<Record<Day, FullDayJa>> = {
+  Mon: "月",
+  Tue: "火",
+  Wed: "水",
+  Thu: "木",
+  Fri: "金",
+  Sat: "土",
+  Sun: "日",
   Intensive: "集中",
   Appointment: "応談",
   AnyTime: "随時",
 };
 
 export const weekdayNum = (day: Day): number => weekdays.indexOf(day);
-export const dayToJa = (day: Day): WeekDayJa => weekdayMap[day] ?? "月";
-export const jaToDay = (day: string): Day =>
-  (Object.keys(weekMap) as (keyof WeekMap)[]).find(
-    (key) => weekMap[key] === day
-  ) ?? "Unknown";
+export const weekNum = (day: Day): number => week.indexOf(day);
+
+export const weekDayToJa = (day: Day): WeekDayJa => weekdayMap[day] ?? "月";
```

Comment:
```suggestion
export const dayToJa = (day: Exclude<Day, "unknown">): WeekDayJa => weekdayMap[day] ?? specialDayMap[day];
```
これで scpecialDay にも対応できると思われる。微妙にまちがっていたらごめん。

Reply:
さっきfullDayJaToDayを作った。

Reply:
ああ、dayToJa の `day` って月～日か。紛らわしい。api.ts の `Day` かと思った。